Instruments/config/2017/Skimmer.cfg

```diff
@@ -104,6 +124,17 @@ merged_output: ttH.root
 file_xs: 0.214 ttHJetToNonbb_M125_TuneCP5.root
 file_xs: 0.293 ttHJetTobb_M125_TuneCP5.root
 
+[ttV]
+merged_output: ttV.root
+file_xs: 0.204 TTWJetsToLNu_TuneCP5.root TTWJetsToLNu_TuneCP5_PSweights.root
+file_xs: 0.406 TTWJetsToQQ_TuneCP5.root
+
+[ttVV]
+merged_output: ttVV.root
+file_xs: 0.008 TTWW_TuneCP5_ext1.root
+file_xs: 0.002 TTWZ_TuneCP5.root
+file_xs: 0.001 TTZZ_TuneCP5.root
+
 [ttX]
```

Comment:
ttX is no longer needed, right?

Reply:
For these, I re-grouped like we discuss on the mattermost chat as:
TTV = TTWJetsToLNu, TTWJetsToQQ
TTVV = TTWW, TTWZ, TTZZ

But there're samples of ttX left out, all the  TTZ: TTZToQQ_TuneCP5.root TTZZ_TuneCP5.root TTZToLLNuNu_M-10_TuneCP5.root TTZToLLNuNu_M-10_TuneCP5_PSweights.root TTZToLL_M-1to10_TuneCP5.root